src/main/java/stringcalculator/Calculator.java

```diff
@@ -0,0 +1,32 @@
+package stringcalculator;
+
+import java.util.regex.Pattern;
+
+public class Calculator {
+
+    public static final int DEFAULT_RESULT = 0;
+    public static final String CUSTOM_REGEX = "//.\\n";
+    static CalculatorNumbers calculatorNumbers;
+
+    public static int calculate(final String operand) {
+        if (operand == null || operand.isEmpty()) {
+            return DEFAULT_RESULT;
+        }
+
+        validateNegativeNumber(operand);
+
+        if (operand.chars().allMatch(Character::isDigit)) {
+            return Integer.parseInt(operand);
+        }
+        calculatorNumbers = new CalculatorNumbers(operand);
+        return calculatorNumbers.sum();
+    }
+
+    private static void validateNegativeNumber(final String operand) {
```

Comment:
메서드 네이밍과 실제로 로직 안에서의 행위가 조금 다르다고 생각되는데요. 
커스텀 연산자의 존재 여부에 대한 검증은 `CalculatorNumbers`에게 위임하고 여기서는 네이밍과 결을 맞춰서 `음수`에 대한 검증만 진행해보면 어떨까요?

Reply:
넵넵 저는 커스텀 연산자가 -일 경우도 생각해보았는데
생각해보니 어떤값이 되었든 음수를 전달하면 예외처리네요
네이밍과는 다르게 일을 2개하고 있는 것도 맞아서 제거 처리했습니다

```suggestion
    private static void validateNegativeNumber(final String operand) {
        if (operand.contains(MINUS_OPERAND)) {
            throw new RuntimeException("음수는 입력할 수 없습니다");
        }
    }
```